__tests__/app.test.js

```diff
@@ -177,3 +178,71 @@ describe("/api/articles", () => {
     });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+    test("GET: 200 - responds with an array of comments for the given article_id", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body: { comments } }) => {
+                expect(comments).toBeInstanceOf(Array);
+                if(comments.length === 0){
+                    return;
+                }
+                comments.forEach(comment => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: expect.any(Number)
```

Comment:
Let's specify here what the article_id is - we know from the endpoint we requested 😄